### PR TITLE
Error hunter1 patch 1

### PR DESCRIPTION
**Pull request by ErrorHunter1**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/femboymc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+     "en",
+  ],
+  "social": {
+    "web": "https://femboy.gmbh",
+    "discord": "https://discord.gg/pfKgrarhMK",
+  },
+  "gamemodes": {
+    "SMP": {
+      "name": "SMP", 
+      "color": "#FF00000,
```
Comment (maaaathis) [inline minecraft_servers/femboymc/manifest.json:19]:
```suggestion
      "color": "#FF00000",
```

File: minecraft_servers/femboymc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "versions": "1.17<*"
+  },
+  "gamemodes": {
+    "CityBuild": {
+      "name": "CityBuild", 
+      "color": "#FF00000,
```
Comment (maaaathis) [inline minecraft_servers/femboymc/manifest.json:25]:
```suggestion
      "color": "#FF00000",
```

File: minecraft_servers/femboymc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "direct_ip": "mc.femboy.gmbh",
+  "server_wildcards": [
+    "%.femboy.gmbh"
+  ],
+  "supported_languages": [
+     "de"
+     "en",
+  ],
+  "social": {
+    "web": "https://femboy.gmbh",
```
Comment (maaaathis) [inline minecraft_servers/femboymc/manifest.json:13]: Its only a maintenance page
image

File: minecraft_servers/femboymc/manifest.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
+  "server_name": "FemBoy",
+  "nice_name": "FemBoyMC",
+  "direct_ip": "mc.femboy.gmbh",
+  "server_wildcards": [
+    "%.femboy.gmbh"
```
Comment (maaaathis) [inline minecraft_servers/femboymc/manifest.json:6]: Doesn't look like a real wildcard
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/femboymc/manifest.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "supported_languages": [
+     "de"
+     "en",
```
Comment (EinsJustinn) [inline minecraft_servers/femboymc/manifest.json:10]:
```suggestion
     "en"
```

File: minecraft_servers/femboymc/manifest.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
+{
+  "server_name": "FemBoy",
+  "nice_name": "FemBoyMC",
+  "direct_ip": "mc.femboy.gmbh",
+  "server_wildcards": [
+    "%.femboy.gmbh"
+  ],
+  "supported_languages": [
+     "de"
```
Comment (EinsJustinn) [inline minecraft_servers/femboymc/manifest.json:9]:
```suggestion
     "de",
```

File: minecraft_servers/femboymc/manifest.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    "CityBuild": {
+      "name": "CityBuild", 
+      "color": "#FF00000,
+      "versions": "1.17<*"
+    }
+  },
+  "discord": {
+    "server_id": 1229059627174592512,
+    "rename_to_minecraft_name": false
+  },
```
Comment (EinsJustinn) [inline minecraft_servers/femboymc/manifest.json:32]:
```suggestion
  }
```

**Review by EinsJustinn — CHANGES_REQUESTED** [review]
update manifest.json

**Review by EinsJustinn — CHANGES_REQUESTED** [review]
update

**Pull request closed without merging**